plugin/pom.xml

```diff
@@ -25,7 +25,7 @@
     <changelist>-SNAPSHOT</changelist>
     <module.name>${project.groupId}.warnings.ng</module.name>
 
-    <analysis-model-api.version>10.13.0</analysis-model-api.version>
+    <analysis-model-api.version>10.14.0-rc601.15a_3a_e9b_5358</analysis-model-api.version>
```

Comment:
You need to set <analysis-model-tests.version>10.13.0</analysis-model-tests.version>. Those tests are not part of the plugin, they are part of the model package.